### ISSUE TITLE
GNN notebook errors on eval

### ISSUE THREAD

**Issue by jalving** [cog-imperial/OMLT#166]
The [GNN notebook](https://github.com/cog-imperial/OMLT/blob/main/docs/notebooks/neuralnet/graph_neural_network_formulation.ipynb) recently starting throwing an error with `print(nn1(torch.tensor(X).float(), torch.tensor(edges)).detach().numpy())` 

Is this something you can take a look at @zshiqiang?

**Comment by zshiqiang** [comment]
I think it is because that a new version of torch_geommetric refactored `Sequential`. I was using `(pooling, "x, None -> x")` to avoid the use of batch, which is not working now. So I add `batch` back as `(pooling, "x, batch -> x")` and it works for both older version and the lastest version. The notebook is prepared and I will pull a request for it.

**Comment by jalving** [comment]
Fixed by #167  Thanks @zshiqiang!
